enterprise/backend/src/metabase_enterprise/metabot_v3/dummy_tools.clj

```diff
@@ -34,11 +34,17 @@
         {:structured-output (-> dashboard (dissoc :collection_id) (assoc :type :dashboard))})
     {:output "dashboard not found"}))
 
+(defn- get-field-values [id->values id]
+  (->
+   (get id->values id)
+   (or (field-values/get-or-create-full-field-values! (t2/select-one :model/Field :id id)))
```

Comment:
How many ids does this get called with? Any point batching the ids into a single `t2/select`?

Reply:
unknown really, it's as many fields don't have field-values which should have. in practice, it can be a 10% subset or lower than all fields, so for now I think on demand is better